apps/dashboard/src/app/[locale]/providers.tsx

```diff
@@ -4,6 +4,7 @@ import { ThemeProvider } from "@/components/theme-provider";
 import { I18nProviderClient } from "@/locales/client";
 import { isDesktopApp } from "@todesktop/client-core/platform/todesktop";
 import { TriggerProvider } from "@trigger.dev/react";
+import { NuqsAdapter } from 'nuqs/adapters/next/app';
```

Comment:
Single quotes here instead of double quotes like other imports?

Reply:
Good catch, fixed.